Src/Utils/FileUtils.cs

```diff
@@ -8,14 +8,25 @@ public class FileUtils : DataUtils
         public FileUtils(string path) : base(path) { }
         private static string GetFileString(string path)
         {
-            byte[] fileBytes = File.ReadAllBytes(path);
-            return StringUtils.ByteArrayToString(fileBytes);
+            if (File.Exists(path))
+            {
+                byte[] fileBytes = File.ReadAllBytes(path);
+                return StringUtils.ByteArrayToString(fileBytes);
+            }
+            else
```

Comment:
No need for else. Move fileBytes out and put it as NULL, and set only if is true...

Reply:
this seam cleaner and should be bit more efficient as the condition is required either way but not the variable allocation